cfme/scripting/quickstart.py

```diff
@@ -112,22 +114,20 @@ def command_text(command, shell):
         return ' '.join(map(quote, command))
 
 
-def call_or_exit(command, shell=False, long_running=False, **kw):
+def run_cmd_or_exit(command, shell=False, long_running=False, **kw):
     try:
         if long_running:
             print(
                 'QS $', command_text(command, shell),
                 '# this may take some time to finish ...')
         else:
             print('QS $', command_text(command, shell))
-        res = subprocess.call(command, shell=shell, **kw)
+        res = subprocess.check_output(command, shell=shell, **kw)
```

Comment:
Good call on switching to check_*() use, but I think we want check_call() here since it waits for the call to complete.  I'm a bit new to both but in checking out the documentation for check_output noticed the difference. Thoughts?

Reply:
If stackoverflow is to be trusted ... this seems to explain the difference in good detail: https://stackoverflow.com/questions/36169571/python-subprocess-check-call-vs-check-output and actually it seems as if check_output would actually be the one that waits longer? :)

Regardless, the reason I'm using it is to get command output. `res = subprocess.call('cmd')` or `res = subprocess.check_call('cmd')` only stores the return code in "res".

check_output is still going to raise if a non-0 exit code is returned, which is all we seemed to care about in this method (if we got a bad exit code, then exit).

Reply:
Ah I see, the difference between the handling of the child/grandchild process output streams wasn't apparent in the python documentation. That was a great explanation of the difference. Thanks!